### PR TITLE
Draw a smiley face when the up arrow is held.

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -11,7 +10,7 @@ use wasm_bindgen::prelude::*;
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
 #[wasm_bindgen]
-extern {
+extern "C" {
```
Comment (cdata) [inline src/lib.rs]: Why "C"?

File: src/lib.rs [hunk 2]
```diff
@@ -21,43 +20,48 @@ pub fn greet() -> String {
...
 
 #[wasm_bindgen]
-pub fn draw() {
-    let document = web_sys::window().unwrap().document().unwrap();
-    let canvas = document.get_element_by_id("canvas").unwrap();
-    let canvas: web_sys::HtmlCanvasElement = canvas
-        .dyn_into::<web_sys::HtmlCanvasElement>()
-        .map_err(|_| ())
-        .unwrap();
+pub fn draw(up: bool, _down: bool, _left: bool, _right: bool) -> Option<bool> {
```
Comment (cdata) [inline src/lib.rs:15]: Params = why?

File: src/lib.rs [hunk 3]
```diff
@@ -21,43 +20,48 @@ pub fn greet() -> String {
...
-        .unwrap()
+        .ok()??
         .dyn_into::<web_sys::CanvasRenderingContext2d>()
-        .unwrap();
+        .ok()?;
+
+    context.clear_rect(0.0, 0.0, canvas.width().into(), canvas.height().into());
+
+    if !up {
+        return Some(true);
```
Comment (cdata) [inline src/lib.rs:30]: 🤷‍♂️

**Review by cdata — APPROVED** [review]

File: src/lib.rs (hunk 1 above)
Comment (rictic) [reply src/lib.rs]: I have no idea. This was an automated change made once I enabled the VSCode plugin

File: src/lib.rs (hunk 3 above)
Comment (rictic) [reply src/lib.rs:30]: 🤷🤷🤷`.ok()?` 🤷🤷🤷

File: src/lib.rs (hunk 2 above)
Comment (cdata) [reply src/lib.rs:15]: I get it now but also complete your thoughts!

File: src/lib.rs (hunk 1 above)
Comment (rictic) [reply src/lib.rs]: Removed since we don't need this anymore anyways

File: src/lib.rs (hunk 2 above)
Comment (rictic) [reply src/lib.rs:15]: Added comment

**Pull request merged**